pandas/io/gbq.py

```diff
@@ -133,13 +267,72 @@ def get_service(self, credentials):
 
         return bigquery_service
 
-    def run_query(self, query):
-        try:
-            from apiclient.errors import HttpError
-            from oauth2client.client import AccessTokenRefreshError
+    def process_http_error(self, ex):
+        # See `BigQuery Troubleshooting Errors <https://cloud.google.com/bigquery/troubleshooting-errors>`__
 
-        except ImportError:
-            raise ImportError('Could not import Google API Client.')
+        status = json.loads(ex.content)['error']
+        errors = status.get('errors', None)
+
+        if errors:
```

Comment:
just raise a generic exception here (you can make 1 custom one if you want). and include the message/reason.